pkg/scheduler/framework/interface.go

```diff
@@ -199,6 +217,7 @@ func (p PluginToStatus) Merge() *Status {
 		}
 		if statusPrecedence[s.Code()] > statusPrecedence[finalStatus.code] {
 			finalStatus.code = s.Code()
+			finalStatus.failedPlugin = s.FailedPlugin()
```

Comment:
Update comment to say that we only keep one that has the most relevant status.

Reply:
done.